--- alsa-utils-1.0.4/alsaconf/alsaconf.in.orig	Fri Mar 19 14:29:23 2004
+++ alsa-utils-1.0.4/alsaconf/alsaconf.in	Thu May 20 14:04:12 2004
@@ -30,9 +30,9 @@
   distribution="debian"
 elif [ -f /etc/mandrake-release ]; then
   distribution="mandrake"
-elif [ -f /etc/redhat-release -a `grep -c "Red Hat" /etc/redhat-release 2>/dev/null` -ne 0 ]; then
+elif [ -f /etc/redhat-release ] && grep -sq "Red Hat" /etc/redhat-release ; then
   distribution="redhat"
-elif [ -f /etc/fedora-release -a `grep -c "Fedora" /etc/fedora-release 2>/dev/null` -ne 0 ]; then
+elif [ -f /etc/fedora-release ] && grep -sq "Fedora" /etc/fedora-release ; then
   distribution="fedora"
 else
   distribution="unknown"
