--- alsa-utils-1.0.16/alsaconf/alsaconf.in~	2008-02-05 11:25:23.000000000 +0200
+++ alsa-utils-1.0.16/alsaconf/alsaconf.in	2008-05-30 14:32:21.779043202 +0300
@@ -300,7 +300,7 @@
   cfgfile="/etc/modules.d/alsa"
 elif [ "$kernel" = "new" ]; then
   if [ -d /etc/modprobe.d ]; then
-    cfgout="/etc/modprobe.d/sound"
+    cfgout="/etc/modprobe.d/sound.conf"
   fi
   cfgfile="/etc/modprobe.conf"
 elif [ "$distribution" = "debian" ]; then
