--- alsa-utils-1.0.20/alsaconf/alsaconf.in.orig	2009-05-06 09:07:24.000000000 +0200
+++ alsa-utils-1.0.20/alsaconf/alsaconf.in	2009-05-17 14:21:33.032777327 +0200
@@ -306,14 +306,14 @@
   cfgfile="/etc/modprobe.conf"
   if [ -d /etc/modprobe.d ]; then
     cfgout="/etc/modprobe.d/50-sound.conf"
-    cfgoldout="/etc/modprobe.d/sound"
+    cfgoldout="/etc/modprobe.d/sound.conf"
     if [ ! -f $cfgout ]; then
 	modver=$(modprobe -V | sed -e's/^module-init-tools version //')
 	case "$modver" in
 	3.[789]*)
 	  ;;
 	*)
-	  cfgout="/etc/modprobe.d/sound"
+	  cfgout="/etc/modprobe.d/sound.conf"
 	  cfgoldout=""
 	  ;;
 	esac
