diff -Nru alsa-utils-0.5.10/aplay/Makefile.am alsa-utils-0.5.10.new/aplay/Makefile.am
--- alsa-utils-0.5.10/aplay/Makefile.am	Sat Nov  4 13:17:20 2000
+++ alsa-utils-0.5.10.new/aplay/Makefile.am	Thu Dec  7 18:15:19 2000
@@ -16,9 +16,9 @@
 	$(LN_S) $< $@
 
 install-exec-hook:
-	rm -f $(bindir)/arecord
+	rm -f $(DESTDIR)$(bindir)/arecord
 	(cd $(DESTDIR)$(bindir) && $(LN_S) aplay arecord)
 
 install-data-hook:
-	rm -f $(mandir)/man1/arecord.1
+	rm -f $(DESTDIR)$(mandir)/man1/arecord.1
 	(cd $(DESTDIR)$(mandir)/man1 && $(LN_S) aplay.1 arecord.1)
